code/modules/mining/surprises/vg.dm

```diff
@@ -87,11 +87,11 @@
 /mining_surprise/alien_nest
 	name="Hidden Nest"
 	floortypes = list(
-		/turf/unsimulated/floor/asteroid=100
+		/turf/simulated/floor/asteroid=100
```

Comment:
Somebody's gonna get angry at this, but eh, it's just for short areas.

Means you can possibly make them into your own little house